src/main/resources/public/index.html

```diff
@@ -362,10 +362,14 @@ <h2 align="center">FIRE</h2>
             </tr>
         </table>
             <table>
-                <td bgcolor="blue" width="50px" height="50px" ></td>
-                <td>miss</td>
-                <td bgcolor="yellow" width="50px" height="50px" ></td>
-                <td>hit</td>
+                <td bgcolor="blue" id="missbg" width="50px" height="50px" ></td>
```

Comment:
make sure the color are changing correctly so that the player will not misunderstand.